### PR TITLE
Jess list events alphabetically

### PR DESCRIPTION
**Pull request by mertzjl91**
Lists events alphabetically. Should show error message if no events are saved to data base. Not sure if that's even necessary so let me know if we should take that out!

### CODE REVIEW

File: src/app/event-list/event-list.component.ts [hunk 1]
```diff
@@ -10,13 +10,20 @@ import { EventService } from '../service/event-service.service';
...
   constructor(private eventService: EventService) { }
 
   ngOnInit() {
     this.eventService.findAll().subscribe(data => {
       this.events = data;
+      this.alphabetizedByName = this.events.sort(function(a, b) {return a.name.localeCompare(b.name)});
     });
   }
 
+  // alphabetizeEvents(eventsArr: Event[]) {
```
Comment (miketully426) [inline src/app/event-list/event-list.component.ts]: Do we need this commented code?

**Review by miketully426 — DISMISSED** [review]

File: src/app/event-list/event-list.component.ts (hunk 1 above)
Comment (mertzjl91) [reply src/app/event-list/event-list.component.ts]: No we don't! I thought I had deleted that function completely at the bottom, but it looks like I missed the first line of it! Will delete!

Comment (mertzjl91) [reply src/app/event-list/event-list.component.ts]: Or the whole thing... whoops!

**Review by aeydie — COMMENTED** [review]
Testing showed working functionality.

File: src/app/event-form/event-form.component.html [hunk 2]
```diff
@@ -29,6 +29,7 @@
               name="location" 
               placeholder="Enter Event Location"
               required #location="ngModel">
+            <div [hidden]="location.pristine || !!location.value" class="alert alert-danger">Location is required</div>
```
Comment (lfallert) [inline src/app/event-form/event-form.component.html:32]: We have alert messages for missing fields for name, description and location. Do we want to include alert messages for the other fields (time, date, etc)? Family friendly checkbox could probably be left as is.

File: src/app/model/event.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 export class Event {
 
-    id: string;
+    id: number;
```
Comment (lfallert) [inline src/app/model/event.ts:3]: Does this need to be changed to eventId?

**Review by lfallert — COMMENTED** [review]
id name in event model class may need to be changed to eventId?

do we want to add alert messages to each field in the event-form.html?

File: src/app/model/event.ts (hunk 3 above)
Comment (lfallert) [reply src/app/model/event.ts:3]: Disregard! I realized that you changed the name of the field on your search-event-by-keyword pull request :)

File: src/app/event-form/event-form.component.html (hunk 2 above)
Comment (aeydie) [reply src/app/event-form/event-form.component.html:32]: Good catch - I didn't even notice that :)

Comment (mertzjl91) [reply src/app/event-form/event-form.component.html:32]: Changes made!

**Review by lfallert — APPROVED** [review]
Tested and working correctly. Events print alphabetically, (symbols and numbers first, followed by letters). I think we can keep the alert message in if there are no events listed in the database (but hopefully there are events always in the database) ha!

**Review by CNichole726 — APPROVED** [review]
Looks good to me!

**Pull request merged**